.github/CODEOWNERS

```diff
@@ -1,3 +1,14 @@
+# Security related
 /Tests/Marketplace/approved_tags.json @demisto/content-leaders @michalgold
 /Tests/Marketplace/approved_usecases.json @demisto/content-leaders @michalgold
-/Packs/CommonScripts/Scripts/SetGridField/* @yaakovi
+
+# Important integrations
+/Packs/Jira/Integrations/JiraV2/* @demisto/content-leaders
+/Packs/ServiceNow/Integrations/ServiceNowv2/* @demisto/content-leaders
+/Packs/EWS/Integrations/EWSv2/* @demisto/content-leaders
+/Packs/Slack/Integrations/Slack/* @demisto/content-leaders
+/Packs/SplunkPy/Integrations/SplunkPy/* @demisto/content-leaders
+/Packs/Active_Directory_Query/Integrations/Active_Directory_Query/* @demisto/content-leaders
```

Comment:
add 
VirusTotal
ArcSight ESM v2

Reply:
maybe QRadar? we did not have a bad experience with it but it is widely used.